### ISSUE TITLE
Install in HACS but not showed up in Integrations

### ISSUE THREAD

**Issue by randykil** [nonsleepr/ha-eufy-security#63]
I install Eufy Security in HACS, but when I search in configuration/Integrations, I can't find Eufy. Any sugestions?

**Comment by thewolf-oz** [comment]
Just confirming you've restarted and also refreshed/cleared-cache on your browser? Sometimes I find both are required after adding to HACS

**Comment by mced76** [comment]
> I install Eufy Security in HACS, but when I search in configuration/Integrations, I can't find Eufy. Any sugestions?

Same issue with the last HA realease. (cleared-cache done)

**Comment by randykil** [comment]
I restarted HA and cleared cashe, still no Eufy in Integrations.

**Comment by remyderuysscher** [comment]
ERROR (SyncWorker_1) [homeassistant.loader] The custom integration 'eufy_security' does not have a valid version key (None) in the manifest file and was blocked from loading. See https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes#versions for more details

**Comment by mced76** [comment]
yes, it works if you add a release number in the /config/custom_components/eufy_security/manifest.json
I add {
  "domain": "eufy_security",
  "name": "Eufy Security",
  "config_flow": true,
  "version": "1.0.0",
...

and after reboot, I can see it in configurations/integrations


Warning, double factor authenticator seems not to be supported

**Comment by remyderuysscher** [comment]
Is this code still maintained? It's been a while since any new releases.

**Comment by randykil** [comment]
This works me too. I am curious of the answer to the question from "remyderuysscher"

**Comment by randykil** [comment]
Damn, i'm running into the next issue. When I want to setup the intergration a login screen apear, but after filling in my username and password nothing happend. Sombody has a clue?

**Comment by mced76** [comment]
> Damn, i'm running into the next issue. When I want to setup the intergration a login screen apear, but after filling in my username and password nothing happend. Sombody has a clue?

Same issue with double factor authenticator. Change to Simple factor in eufy settings and it works!

**Comment by thewolf-oz** [comment]
I had the same issue... ended up installing manually (copy folder over the custom_components folder) and it showed in integrations. I think there was a comment in one of the blogs or forum entries I saw regarding single factor authentication so I setup a second account with super complex password and shared my devices (as per the recommendation in that entry) and that's what I have in HA

**Comment by profucius** [comment]
Just a tip for anyone running into this issue: Open manifest.json, if there is already a version 0.3.0 at the bottom, just replace that with 1.0.0, save and reboot. It will work. I don't know why it fails with the default version number.

**Comment by gijsprins93** [comment]
Even with the tips of setting the version, it does not work for me... It does not show up in integrations

**Comment by rvdbeek** [comment]
The tip clearing the cache in my chrome browser did the trick. Now I see the integration again. 
Ctrl+Shift+R is a keyboard shortcut used to perform a hard reload of a web page in Google chrome

**Comment by QziP22** [comment]
Can confirm the instructions provided by profucius do indeed work. Eufy wasn't showing up in my integrations either, adding the version number solved that.

**Comment by D-Cox** [comment]
After I went through all of the suggestions above, I finally got the integration to install. I entered the login information, and everything looks fine. But I don't see the device. It never seems to be discovered. I'm not using the dual authentication. Any one have any thoughts?

**Comment by enitalp** [comment]
I found out, two thing, i had this version with no version in it, and it never updated itself, no indication it was up to date. deleting it restart, reinstall, oh , a brand new version, with a manifest with version # ! 

Second even with that, no luck still, no Eufy in the integration , messup with files, restart, check, etc..
But that time i left my HA alone for 5 mins, and went again in integration and eufy was there. 
So it's not there at startup, but if you wait a little it will appear !
